xarray/tests/test_dask.py

```diff
@@ -178,6 +178,20 @@ def test_binary_op(self):
         self.assertLazyAndIdentical(u + u, v + v)
         self.assertLazyAndIdentical(u[0] + u, v[0] + v)
 
+    def test_binary_op_bitshift(self):
```

Comment:
```suggestion
    def test_binary_op_bitshift(self) -> None:
```
Add None so mypy also checks the test.